src/cli/commands/build.ts

```diff
@@ -14,6 +14,7 @@ const BUILD_ARGS = {
   '--stage': String,
   '--entrypoint': String,
   '-e': '--entrypoint',
+  '--no-bundle': Boolean,
```

Comment:
Not sure how to call the new output. It's not really a bundle, but then, what it is?